src/index.ts

```diff
@@ -39,11 +46,7 @@ export default class HuddlyDeviceAPIUSB implements IHuddlyDeviceAPI {
   }
 
   async getValidatedTransport(device: usb.Device): Promise<ITransport> {
-    if (
-      [HuddlyHEX.GO_PID, HuddlyHEX.L1_PID, HuddlyHEX.S1_PID, HuddlyHEX.BASE_PID].includes(
-        (device as any as UsbDevice).productId
-      )
-    ) {
+    if (defaultPidsToIgnore.includes((device as any as UsbDevice).productId)) {
```

Comment:
Hvorfor type as any as UsbDevice?

Reply:
Copy paste, probably can remove, but I would rather do it as a bigger overhaul.